### PR TITLE
Set default waterfall colors static viz

### PR DESCRIPTION
**Pull request by howonlee**
Pursuant to issue #20332 - it was blanking out without settings set before

### CODE REVIEW

**Comment by howonlee** [comment]
@metabase-bot run visual tests

**Comment by howonlee** [comment]
looks like you can't set the show total to false on this one, will fix that too

File: src/metabase/pulse/render/body.clj [hunk 1]
```diff
@@ -795,12 +795,15 @@
         render-fn      (if (isa? (-> cols x-axis-rowfn :effective_type) :type/Temporal)
                          js-svg/timelineseries-waterfall
                          js-svg/categorical-waterfall)
+        show-total     (if (nil? (:waterfall.show_total viz-settings))
+                         true
+                         (:waterfall.show_total viz-settings))
```
Comment (noahmoss) [inline src/metabase/pulse/render/body.clj:800]: This is just making `true` the default, right? You could just do `(:waterfall.show_total viz-settings true)`

**Review by noahmoss — APPROVED** [review]

File: src/metabase/pulse/render/body.clj (hunk 1 above)
Comment (howonlee) [reply src/metabase/pulse/render/body.clj:800]: I've seen `nil` as actual set value in it once or twice, for which this semantics is different, but that's legit another bug, too, lol

**Pull request merged**